CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.9.0 (2020-03-13)
+
+- Added the format component which allows displaying rich content within a tokenized string. [#89](https://github.com/blackbaud/skyux-layout/pull/89)
```

Comment:
Will editing to "..which displays rich content" change meaning? If it doesn't, that's my recommended edit.

Reply:
I made the suggested change. Thanks!